src/app/services/validators/customValidators.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { AbstractControl, ValidatorFn } from '@angular/forms';
+import { TestBed, inject } from '@angular/core/testing';
+import { CustomValidators } from './customValidators';
+
+describe('CustomValidators', () => {
+
+    let customvalidator : CustomValidators
+
+    beforeEach(() => {
+      TestBed.configureTestingModule({
+        imports: [],
+        providers: []
+      });
+      customvalidator = new CustomValidators;
+    });
+
+
+  
+    it('should be created', () => {
+      expect(customvalidator).toBeTruthy();
+    });
+  
+    // describe('dateLessThan', () => {
```

Comment:
Commented out code should be remove. 

Also there should probably be some tests in here :-)